elements/bulbs-video/components/cover.test.js

```diff
@@ -51,7 +51,7 @@ describe('<bulbs-video> <Cover>', function () {
       );
     });
 
-    it('renders a CroppedImage', () => {
+    it('renders an image', () => {
```

Comment:
As you may know there are problems with using CroppedImage in bulbs-elements.

The local image.js takes over and starts rendering the wrong images.